CONTRIBUTING.md

```
@@ -27,6 +79,10 @@ Thank you for contributing to Vertex Forager!
   - Rust: `cargo install lychee --locked`
   - Run: `pre-commit run lychee -a`
 
+### Labels policy (overview)
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)